policyengine_us/parameters/gov/states/la/tax/income/exemptions/personal/blind.yaml

```diff
@@ -0,0 +1,15 @@
+description: Louisiana provides an exemption of this amount for blind filers.
+values:
+  2014-01-01: 1_000
+
+metadata:
+  reference:
+    - title: Louisiana Laws Revised Statutes, Title 47 — Revenue and taxation, RS 47:79 — Credits of individuals against net income §79, A, (2)
+      href: https://www.legis.la.gov/legis/Law.aspx?d=102518
+    - title: Louisiana Individual Income Tax
+      href: https://revenue.louisiana.gov/individualincometax
```

Comment:
this seems to only reference the 1,000 for Nonresident and Part-Year Resident Individual Income Tax and Nonresident Athlete Individual Income Tax

Reply:
if using this source add the specific header of the section that mentions the parameter

Reply:
I dont think this one does

Reply:
check for all yaml files